kedro/framework/cli/starters.py

```diff
@@ -288,11 +444,12 @@ def _get_cookiecutter_dir(
                 f" Specified tag {checkout}. The following tags are available: "
                 + ", ".join(_get_available_tags(template_path))
             )
-        official_starters = sorted(_STARTER_ALIASES)
-
+        official_starters = sorted(_OFFICIAL_STARTERS_SPECS)
```

Comment:
How does this implement the `sort`? I see you have `order=True` but what does that actually generate for the comparison methods?

Reply:
See another comments I have for Merel, but it's quite intuitive. You can treat it like a namedtuple basically.